site/sfguides/src/connectors_github_python/github_python_connector.md

```diff
@@ -100,41 +104,40 @@ The connector configuration is split into two domains:
 - resource level configuration
 
 ### Configuring connector
-Connector level configuration lives in `STATE.APP_CONFIGURATION` table. This table is meant to keep information
-like API integration name used by the connector or secret name used for authentication to GitHub.
+Connector level configuration lives in the `STATE.APP_CONFIGURATION` table. This table is meant to keep information
+like API integration name used by the connector and secret name used for authentication in GitHub.
 
 The table is a key-value table.
 
 ### Ingestion resource configuration
-Next to global configuration the connector also stores configuration for every enabled resource. In case of this example
+Next to the global configuration the connector also stores a configuration for every enabled resource. In case of this example
 a single resource is represented as `org_name/repo_name`.
 
 The table is a key-value table.
 
 ## Connector state
 Duration: 2
 
-Apart from configuration table, the connector uses `STATE.APP_STATE` table for persisting state of ongoing ingestions.
+Apart from configuration table, the connector uses `STATE.APP_STATE` table to persist a state of any ongoing ingestion.
 
-The state table is key-value table with additional timestamp column. To improve the perfomance the table is an append only table.
+The state table is a key-value table with an additional timestamp column. To improve the performance, the table is an append only table.
 This means that creating or updating a value inserts a new row into the table. Reading a key is done by retrieving the
-newest row with given key.
+newest row with the given key.
 
 ## External access
 Duration: 2
 
-Data from Github is ingested using external access capabilities of Snowflake.
+Data from GitHub is ingested using external access capabilities of Snowflake.
 
 ### Direct external access
 
-If you are part of direct external access Private Preview you can continue to next step. If not then please
-follow additional necessary steps from the next paragraph.
+Direct external access is a public preview feature of Snowflake.
```

Comment:
would be good to add link to https://docs.snowflake.com/en/developer-guide/external-network-access/external-network-access-overview

Reply:
done